config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/ListenPortEntityBuilder.java

```diff
@@ -42,7 +47,7 @@
 
     private static final Map<String, ListenPort> DEFAULT_PORTS = unmodifiableMap(createDefaultListenPorts());
     private static final String USES_TLS = "usesTLS";
-    private static final Integer ORDER = 800;
+    private static final Integer ORDER = 1200;
```

Comment:
What is integer change for?

Reply:
ListenPort builder has to be invoked after the private key builder, so it finds the key into the bundle.